app/controllers/devise/passwords_controller.rb

```diff
@@ -47,7 +47,7 @@ def update
       respond_with resource, location: after_resetting_password_path_for(resource)
     else
       set_minimum_password_length
-      respond_with resource
+      respond_with resource, status: :unprocessable_entity
```

Comment:
Were you having any issue with responders alone here, to have to enforce a status?

Reply:
@baarkerlounger made that change in https://github.com/ghiculescu/devise/commit/eef20a326075b3f36df7f8b2a35c6f5ebe4b4f6b

The discussion is here https://github.com/heartcombo/devise/pull/5340#issuecomment-1029004041, I don't really remember any context beyond that.

Reply:
Gotcha, thanks. I think those should be handled by default with responders now :+1: